scripts/find.rb

```diff
@@ -0,0 +1,3 @@
+People.find_by age: > 35 OR title: 'Queen of Meereen and Mother of Dragons'
```

Comment:
This syntax is close, but `Person` is the name of your model, and you should use `.where` when you are looking for more than one result